.env.example

```diff
@@ -57,10 +57,10 @@ DIGITAL_OCEAN_WORKSPACE_WEBHOOK=DIGITAL_OCEAN_WORKSPACE_WEBHOOK
 SUPPORT_EMAIL_WORKSPACE_WEBHOOK=SUPPORT_EMAIL_WORKSPACE_WEBHOOK
 WORK_LOGS_URL=WORK_LOGS_URL
 WORK_LOGS_API_SECRET=WORK_LOGS_API_SECRET
-ADMIN_DM_ID=ADMIN_DM_ID
-OPS_DM_ID=OPS_DM_ID
-ENGINEERING_DM_ID=ENGINEERING_DM_ID
-BIZDEV_DM_ID=BIZDEV_DM_ID
+MISSING_WORK_LOGS_ADMIN_WORKSPACE_WEBHOOK=MISSING_WORK_LOGS_ADMIN_WORKSPACE_WEBHOOK
```

Comment:
Based on what we discussed yesterday, while the OUs definition gets finished, we could adjust this to just send every notification to the same space, we could name it something like `MISSING_WORK_LOGS_WORKSPACE_WEBHOOK`

What do you think?